source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -235,13 +235,15 @@ FilterHeadersStatus Filter::decodeHeaders(RequestHeaderMap& headers, bool end_st
     decoding_state_.setCompleteBodyAvailable(true);
   }
 
-  if (!decoding_state_.sendHeaders()) {
-    ENVOY_LOG(trace, "decodeHeaders: Skipped");
-    return FilterHeadersStatus::Continue;
+  FilterHeadersStatus status = FilterHeadersStatus::Continue;
+  if (decoding_state_.sendHeaders()) {
+    status = onHeaders(decoding_state_, headers, end_stream);
+    ENVOY_LOG(trace, "onHeaders returning {}", static_cast<int>(status));
   }
```

Comment:
Probably add a else {
 ENVOY_LOG(trace, "decodeHeaders: Skipped");
}
here to retain the original log, which is very helpful in debugging.